src/lang/base/Syntax.ml

```diff
@@ -202,6 +202,134 @@ type literal =
                CPSMonad.t)
 [@@deriving sexp]
 
+(* Builtins *)
+type builtin =
+  | Builtin_eq
+  | Builtin_concat
+  | Builtin_substr
+  | Builtin_strlen
+  | Builtin_to_string
+  | Builtin_blt
+  | Builtin_badd
+  | Builtin_bsub
+  | Builtin_to_uint256
+  | Builtin_sha256hash
+  | Builtin_keccak256hash
+  | Builtin_ripemd160hash
+  | Builtin_to_bystr
+  | Builtin_schnorr_verify
+  | Builtin_ec_gen_key_pair (* in gas coster only *)
```

Comment:
It has been decided that to remove these four builtins because they deal with private keys and it's unlikely these will be exposed in the future, but we would like to keep a reminder in the source code.

Let's comment these four builtins out, move them to the end of the definition and remove the corresponding cases from the gas coster and the cashflow checker. Also, could you please add a comment linking to this PR, perhaps to this very comment?